src/atoms/Space/Space.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text } from '@storybook/addon-knobs';
+import Text from '../Text';
+import Space from './Space';
+
+storiesOf('Space', module)
+  .addParameters({
+    component: Space,
+  })
+  .add('default', () => (
+    <Space margin={[1, 1]} padding={[2, 2]}>
+      <Text as="span">{text('Display Text', 'The quick brown fox jumps over the lazy dog ')}</Text>
```

Comment:
Don't think size knob for Text component is useful for Space stories.
Better have knobs around Space component itself.

Reply:
Didn't get this.

Reply:
I meant that we could add stories showing different variations of Space. It's fine since it's a very simple component. Will close this. 👍